src/main/java/nextstep/ladder/domain/Direction.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.ladder.domain;
+
+import java.util.Arrays;
+
+public enum Direction {
+    RIGHT((left, right) -> {
+        return !left && right;
+    }),
+
+    LEFT((left, right) -> {
+        return left && !right;
+    }),
+
+    PASS((left, right) -> {
+        return !left && !right;
+    });
+
+    private final DirectionOperation directionOperation;
+
+    Direction(DirectionOperation directionOperation) {
+        this.directionOperation = directionOperation;
+    }
+
+    public static Direction getDirection(boolean left, boolean right) {
+        return Arrays.stream(Direction.values())
+                .filter(direction -> direction.directionOperation.getDirection(left, right))
+                .findFirst()
+                .orElse(null);
+    }
+
+    public static boolean isLeft(Direction direction) {
+        return direction == Direction.LEFT;
+    }
```

Comment:
```suggestion
    public boolean isLeft() {
        return this == Direction.LEFT;
    }
```
굳이 클래스 메서드가 아니라 위와 같이 인스턴스 메서드로 구현해도 되지 않을까?